docs/cimpl6ToolingReference.md

```diff
@@ -498,19 +497,39 @@ For many users, especially those with experience in object-oriented modeling, th
 
 ### Data Dictionary Export
 
-The Data Dictionary is a Microsoft Excel file containing a simplified, flattened list of model `Elements` and value sets, extracted from the FHIR profiles. The spreadsheet has five tabs:
+The Data Dictionary is a Microsoft® Excel® file containing a list of model `Elements` and value sets extracted from the FHIR profiles. The spreadsheet has four tabs:
 
-* The _Key_ tab contains a key to the columns in the other tabs,
-* The _Profiles_ tab contains a list of the primary profiles and their descriptions,
-* The _Data Elements_ tab contains a list of [**MustSupport**](https://www.hl7.org/fhir/conformance-rules.html#mustSupport) and required elements associated with those profiles,
-* The _Value Sets_ tab contains a list of value sets used in the IG, and their descriptions,
-* The _Value Set Details_ tab contains a list of the codes and/or implicit definitions in those value sets.
+* The _Profiles_ tab contains a list of the profiles, with descriptions, that are defined in the IG,
+* The _Data Elements_ tab contains a list of profiles and elements, as defined in the IG [Content Profile](#content-profile-file). If a **MustSupport** element is bound to an externally defined value set, the value set URL is populated on this tab.
+* The _Value Sets_ tab contains a list of value sets defined in the IG, that are bound to **MustSupport** elements, and their descriptions, <!-- internal note: any externally defined value sets bound to a must support element are added to the list of "value sets to put into the value set tab", however since the exporter will fail to find a value set in the spec associated with the url, no row is written to the value set tab -->
+* The _Value Set Details_ tab contains a list of the codes and/or implicit definitions in those value sets (if available).
 
 The Data Dictionary can be useful both to clinicians and implementers who don't need the structural details of the FHIR profiles, but want the contents in list form. For many reviewers and contributors, this can be an easy-to-use format to facilitate discussion and feedback.
 
+<!-- CC to do: Location of planned reference to DD details. Appendix: Data Dictionary Details documents the four tab columns. MT to provide -->
+
+### Creation Steps
+The data dictionary is created automatically whenever a **MustSupport** element is defined in the Content Profile, or a profile is created as part of the IG.
```

Comment:
The data dictionary is created by default. If the user doesn't want to produce the DD, there is an option in the command line interface skip (-s) option. Adding `-s data-dict` it should skip the data dictionary.

Reply:
OK, thanks. Will add.